common-util/AbiAndAddresses/contributeManager.js

```diff
@@ -0,0 +1,489 @@
+export const CONTRIBUTE_MANAGER_ADDRESS_BASE = '0x8394a65f5096b0708428e3ed86cb874e1f3fd724';
```

Comment:
is this on tenderly fork?
https://basescan.org/address/0x8394a65f5096b0708428e3ed86cb874e1f3fd724

Reply:
yes... seems this didn't work :DD
<img width="867" alt="image" src="https://github.com/user-attachments/assets/0387606e-256f-4a09-8ae0-a6620cc8b055">

Reply:
i just saw the purple and thought i was cool

Reply:
ahahah